src/components/choice-tabs/ChoiceTabs.scss

```
@@ -218,11 +218,12 @@
     scroll-snap-type: x mandatory;
     -webkit-overflow-scrolling: touch;
     scroll-padding: 0 16px;
-    padding-bottom: 6px;
-    margin-left: -16px;
-    margin-right: -16px;
-    padding-left: 16px;
-    padding-right: 16px;
+    // overflow-x:auto implicitly turns overflow-y into a clip. The active
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)